doc/nrf/releases/release-notes-1.6.0.rst

```diff
@@ -0,0 +1,651 @@
+  * Fixed an issue in :ref:`bt_mesh_light_ctrl_srv_readme` by always setting the transition time to a Fade Standby Manual state time when a Light Off event is received.
+  * Fixed an issue by reporting maximum remaining time for all components for CTL state transition time when GET is processed.
+  * Fixed an issue where a deleted :ref:`bt_mesh_scene_srv_readme` did not delete all its pages from the file system.
+  * Fixed an issue where Sensor Threshold was trimmed and an invalid value was calculated.
+  * Updated :ref:`bt_mesh_scheduler_srv_readme` to no longer extend :ref:`bt_mesh_scene_srv_readme`, but they must be present on the same element.
+  * Moved the configuration settings for acknowledged messages into Kconfig to make them public.
+  * Fixed an issue where an Occupancy On event did not transition to a Fade On state even if Occupancy Mode is disabled.
+  * Added a flag to :ref:`bt_mesh_onoff_srv_readme` to skip Default Transition Time on Generic OnOff Set (Unack) messages.
+  * Fixed an issue by correcting the bindings between the Generic OnOff state and the Light OnOff state.
+  * Fixed an issue by clearing the internal sum in the proportional-integral (PI) feedback regulator when entering the OFF state of the state machine.
+  * Fixed an issue where :ref:`bt_mesh_lightness_srv_readme` could accidentally disable :ref:`bt_mesh_light_ctrl_srv_readme` before it was started.
+  * Fixed an issue by publishing the Light Lightness Status message on startup even if the OnPowerUp state is OFF.
+  * Fixed issues by publishing the Light OnOff Status when disabling and restoring the Light LC state.
+  * Fixed an issue where temperature and range should be within a valid default range for the :ref:`bt_mesh_light_temp_srv_readme`.
+  * Removed sensor type ``BT_MESH_SENSOR_DELTA_DISABLED`` as it is removed from specification.
+  * Replaced ``struct bt_mesh_model_ack_ctx`` with ``struct bt_mesh_msg_ack_ctx`` from :ref:`zephyr:bluetooth_mesh_msg`.
+
+Matter (Project CHIP)
+---------------------
+
+* Project CHIP has been officially renamed to `Matter`_.
+* Added:
+
+  * New user guide about :ref:`ug_matter_configuring`.
+
+* Updated:
+
+  * Renamed occurrences of Project CHIP to Matter.
+  * Updated the Matter fork in the |NCS| to the revision mentioned in the Matter (Project CHIP) submodule section below.
+
+Thread
+------
+
+* Added support for the following Thread 1.2 features:
+
+  * Domain Unicast Addresses
+  * Multicast Listener Registration
+  * Backbone Router (Thread Network side only)
+  * Link Metrics
+  * Coordinated Sampled Listening (CSL)
+
+  Link Metrics and CSL are supported for the nRF52 Series devices.
+* NCP sample renamed to :ref:`Co-processor <ot_coprocessor_sample>`, with added support for the :ref:`thread_architectures_designs_cp_rcp` architecture.
+
+Zigbee
+------
+
+In this release, Zigbee is supported for development and should not be used for production.
```

Comment:
@gkafel please confirm this is correct

Reply:
PMT wanted indication what is the Zigbee release which could be used for production (namely: NCS 1.5.1)

Reply:
it can be found in the next line